### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by AddoSolutions**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,41 +34,29 @@ And thank you for your help!
 * [1848 Ventures](https://www.1848ventures.com/)
 * [216digital](https://216digital.com/)
 * [216 Software](http://216software.com/)
-* [2strand](https://www.2strand.com/)
 * [33 Mile Radius](https://www.33mileradius.com/)
 * [41n.io](http://www.41n.io/)
 * [42connect](https://www.42connect.com/)
 * [7signal](http://7signal.com/)
-* [ABB](https://new.abb.com/jobs/center#JobCountry=US&JobCity=Cleveland&JobFunction=any&JobRole=any&JobText=)
```
Comment (mrfright) [inline README.md:42]: This can be updated to this

https://careers.abb/global/en/search-results?keywords=Cleveland

File: README.md [hunk 2]
```diff
@@ -34,41 +34,29 @@ And thank you for your help!
...
 * [Aero Fluid Products](http://www.aerofluidproducts.com/)
 * [Affinity](http://www.affinityit.com/)
 * [Agile Blue](https://agileblue.com)
-* [Agility Business Solutions](http://www.agilitybiz.net/)
 * [Allied Health Media](http://www.alliedhealthmedia.com/)
-* [alltech](http://www.alltechmedusa.com/Homepage.html)
-* [Alpha Net Consulting](http://www.anetcorp.com/careers/) - with jobs like [Firmware/Embedded Engineer](https://www.dice.com/jobs/detail/Firmware%26%2347Embedded-Engineer-Alpha-Net-Consulting-LLC-Mayfield-Heights-OH-44124/10195458/506879) on Dice
 * [American Greetings](http://www.americangreetings.com/)
 * [AmTrust](https://amtrustfinancial.com/careers/hiring-process)
-* [Applied Industrial Technologies](http://web.applied.com/base.cfm?page_id=4658)
```
Comment (mrfright) [inline README.md:55]: This can be updated to this

https://jobs.applied.com/us/en

File: README.md [hunk 3]
```diff
@@ -34,41 +34,29 @@ And thank you for your help!
...
 * [Apteryx](http://www.apteryx.com/)
 * [Arhaus](http://www.arhaus.com/)
-* [Aronetics](https://www.aronetics.com/about-us/)
```
Comment (mrfright) [inline README.md]: This can be updated to this

https://www.aronetics.com/company/join-us/

File: README.md [hunk 4]
```diff
@@ -34,41 +34,29 @@ And thank you for your help!
...
 * [Arras Keathley](http://www.arraskeathley.com/)
-* [Artemis Intelligence](https://www.artemisintel.io/), their [Workable page](https://artemisintel.workable.com/)
```
Comment (mrfright) [inline README.md:60]: Looks like they're now Aperture, could update to that.

https://aperture.ai/

Wait a minute. Aperture. AI. Cleveland.

https://half-life.fandom.com/wiki/Borealis?file=Borealis_plans_01.png

File: README.md [hunk 5]
```diff
@@ -34,41 +34,29 @@ And thank you for your help!
...
 * [ASMGi](https://www.asmgi.com/)
-* [Associated Software Consultants](https://www.asconline.com/ftp/) - makers of [Power Lender](https://www.powerlender.com/)
```
Comment (mrfright) [inline README.md:62]: Power lender link still works, change to that.

File: README.md [hunk 6]
```diff
@@ -77,11 +65,9 @@ And thank you for your help!
 * [Bear VAI Technology](http://bearvai.com/inc/our_team/career-opportunities/)
 * [because I said I would](http://becauseisaidiwould.com/)
 * [Beegit](https://www.beegit.com/)
-* [Bennett Adelson](https://www.bennettadelson.com/) - their [Taleo Job Search](http://chm.tbe.taleo.net/chm03/ats/careers/jobSearch.jsp?org=BENNETTADELSON&cws=39)
```
Comment (mrfright) [inline README.md:80]: Their site works just remove or update their Taleo link.

File: README.md [hunk 7]
```diff
@@ -99,7 +85,6 @@ And thank you for your help!
 * [BudgetDumpster](http://www.budgetdumpster.com/)
 * [Buyerquest](http://www.buyerquest.com/)
 * [Capricorn Systems](http://www.capricornsys.com/jobs.html) - jobs like [Embedded C/C++ Programmers](https://www.dice.com/jobs/detail/c8cec17d423b240f2a7eda3b71a0df65?src=32&CMPID=AG_ZR_PD_JS_US_OG_RC_&utm_campaign=Advocacy_Ongoing&utm_medium=Aggregator&utm_source=ZipRecuiter&rx_campaign=ziprecruiter51&rx_group=100954&rx_job=capricor%2Fc8cec17d423b240f2a7eda3b71a0df65&rx_medium=cpc&rx_source=ziprecruiter&zip_cid=53898569-12fd-48d0-9cfa-6c38a7920755) on Dice
-* [Cardinal Commerce](https://www.cardinalcommerce.com/careers)
```
Comment (mrfright) [inline README.md]: This link works, there was just a cloudflare page first.

File: README.md [hunk 8]
```diff
@@ -110,7 +95,6 @@ And thank you for your help!
 * [ClickShake games](http://www.clickshakegames.com/)
 * [Cloud CMS](https://www.cloudcms.com/careers.html)
 * [CodeRed](https://www.coderedcorp.com/)
-* [Codesummit](https://codesummit.com/company/careers/)
```
Comment (mrfright) [inline README.md:113]: Site still there, just not the careers page.

File: README.md [hunk 9]
```diff
@@ -166,10 +146,7 @@ And thank you for your help!
 * [Everkey](https://everykey.com/)
 * [Everstream](http://everstream.net/)
 * [Excellware](http://www.excellware.com/)
-* [ExchangeBase](https://www.exchangebase.com/)
 * [Expand Interactive](http://www.expandinteractive.com/)
-* [Expedient](https://www.expedient.com/company/careers-opportunities/)
```
Comment (mrfright) [inline README.md:171]: Career page moved here

https://expedient.com/careers/

File: README.md [hunk 10]
```diff
@@ -239,19 +209,15 @@ And thank you for your help!
...
-* [Knowvent](http://www.knowvent.com/) - their [API](https://github.com/Knowvent/), [another site of theirs](http://blog.iteedee.com/)
 * [Komae](https://www.mykomae.com/)
-* [Konica Minolta](http://konicaminoltaus.jobs/jobsearch/#All~Job~Categories|S::OH||d-ASC|1)
 * [LayerZero](http://www.layerzero.com/index.html)
 * [Lazorpoint](http://www.lazorpoint.com/)
 * [Lazurite](https://lazurite.co/)
 * [LeadLift](https://www.leadlift.com/)
 * [Leap Amabassadors Community](https://leapambassadors.org/)
 * [Level Seven](http://www.lvlsvn.com/)
-* [LightSpeed Hosting](https://lightspeedhosting.com/)
```
Comment (mrfright) [inline README.md]: This link worked for me

File: README.md [hunk 11]
```diff
@@ -279,14 +245,11 @@ And thank you for your help!
 * [Mindful](http://getmindfully.com/)
 * [Mobile Defense](https://www.mobiledefense.com/)
 * [MobyMax](http://www.mobymax.com/)
-* [Modis](http://www.modis.com/job-seekers/it-jobs/?k=&l=cleveland%2c+oh&xy=41.4993%2C-81.6944&display=10) - jobs like [c embedded rtos](https://www.dice.com/jobs/detail/c-embedded-rtos-Systems-Pros-Inc.-Cleveland-OH-44114/cxmodbma/300916) on Dice
```
Comment (mrfright) [inline README.md:282]: They're now Akkodis, here's the career search page to update

https://www.akkodis.com/en-us/careers/job-results?CityFilter=Akron,Cleveland

File: README.md [hunk 12]
```diff
@@ -303,51 +266,38 @@ And thank you for your help!
...
 * [Onvoy](http://www.onvoy.com/)
-* [Osmisys](http://www.osmisys.com/)
 * [OverDrive](https://www.overdrive.com/) - their [careers page](https://company.overdrive.com/careers/)
-* [Ownum](https://ownum.io/) - blockchain company started by car salesman that started Blockland
   * [CHAMPTitles](https://www.champtitles.com/) - car titles on the blockchain
-* [Palnar](http://palnar.com/jobs/) - with jobs like [Firmware Engineer](https://www.dice.com/jobs/detail/Firmware-Engineer-Palnar-INC-Mayfield-Heights-OH-44124/10217801/2018%26%23451176) on Dice
 * [Pandata](http://pandata.co/about-us/careers/)
 * [Paragon](http://www.paragon-inc.com/)
 * [Parker Hannifin](http://parker.com) - their [job search](http://parkercareers.ttcportals.com/search/jobs/within/50/miles/of/41.49932/-81.69436050000002?radius=41.49932%2C-81.69436050000002%2C40&ns_location=Cleveland%2C+OH%2C+USA)
-* [Park Place Technologies](http://www.parkplacetechnologies.com/) - recently bought [Axentel](http://axentel.com/)
```
Comment (mrfright) [inline README.md:325]: This link is still good.

Comment (mrfright) [reply README.md:325]: ah, axentel link isn't good.

File: README.md [hunk 13]
```diff
@@ -303,51 +266,38 @@ And thank you for your help!
...
 * [PCC Airfoils](http://www.pccairfoils.com/business_units/mentor/)
 * [PCRecruiter](https://www.pcrecruiter.net/) - by Main Sequence Technology
-* [PC Parts Unlimited](http://www.pcpartsohio.com/)
 * [PDP Tax Service](http://pdptax.com/)
 * [Pervasive Path](http://pervasivepath.com/)
 * [Phenom](http://www.phenom.co/) - possibly no longer active, that site is an empty Heroku app and their Facebook page hasn't been active for almost a year
 * [Philips Medical](http://www.healthcare.philips.com/us_en/)
 * [pixsi](http://www.pixsi.com/)
 * [Playaway](https://playaway.com/life-at-playaway/#join-us)
-* [Pleasant Valley Corporation](http://www.pleasantvalleycorporation.com/employment/) - their [careers page](https://careers.pvcconnect.com/Careers.aspx)
```
Comment (mrfright) [inline README.md:337]: Pleasant valley link is good, maybe not that pvcconnect link.

File: README.md [hunk 14]
```diff
@@ -303,51 +266,38 @@ And thank you for your help!
...
 * [PNC Financial](https://www.pnc.com/) - Strongsville, W130th Operations, Highland Hills - [Careers Page](https://sjobs.brassring.com/TGnewUI/Search/Home/Home?partnerid=15783&siteid=5130#Location(s)=OH%20-%20Strongsville,OH%20-%20Cleveland,OH%20-%20Highland%20Hills&keyWordSearch=&locationSearch=)
 * [Pointe Blank Solutions](http://www.pointeblank.net/)
 * [PreEmptive Solutions](https://www.preemptive.com/)
 * [Prentke Romich Company (PRC)](http://www.prentrom.com/)
 * [Proformex](https://www.proformex.com/careers)
-* [Progressive](https://www.progressive.com/)
```
Comment (mrfright) [inline README.md:343]: Pretty sure Progressive is still around.

File: README.md [hunk 15]
```diff
@@ -357,20 +307,15 @@ And thank you for your help!
 * [Renovo](http://renovo1.com/)
 * [Resolvit](http://careers.resolvit.com/Home/countryId/1/stateId/OH)
 * [Revenue Conduit](http://revenueconduit.com/)
-* [RIDGID](https://www.ridgid.com/us/en/job-dot-net-mvc-developer)
```
Comment (mrfright) [inline README.md:360]: Ridgid is still in Elyria

File: README.md [hunk 16]
```diff
@@ -470,56 +396,43 @@ And thank you for your help!
 * [WeCanCodeIT](https://wecancodeit.org/)
 * [Cleveland Codes](http://www.clevelandcodes.org/) - [tri-c site](http://www.tri-c.edu/programs/information-technology/cleveland-codes/index.html)
 * [Tech Elevator](http://www.techelevator.com/)
-* [Case Western Reserve University Bootcamp](https://codingbootcamp.case.edu/)
```
Comment (mrfright) [inline README.md:473]: They still have a bootcamp

https://bootcamp.case.edu/coding/

File: README.md [hunk 17]
```diff
@@ -470,56 +396,43 @@ And thank you for your help!
...
 * [Soulcraft woodshop](http://soulcraftwoodshop.com/), also see [Soulcraft CLE](http://soulcraftcle.org/)
 * [Women Who Code Cleveland](https://www.womenwhocode.com/cleveland)
 * [Cleveland Locksport](https://twitter.com/CLELocksport)
-* [UXPA Cleveland](https://www.uxpacleveland.org/)
-* [Flux Makerspace](http://www.fluxmakerspace.com/)
-* [Studio How-To](https://www.studiohow-to.com/)
 * [We Make the Internet](https://www.meetup.com/make-the-internet/)
 * [Fresh Lab](http://www.refreshcollective.org/freshlab/) - makerspace for music, clothing
 * [CoolTech Challenge](https://mobile.twitter.com/RITECoolTech) - their site not up?
-* [Northern Ohio Amateur Radio Society](https://www.noars.net/)
```
Comment (mrfright) [inline README.md]: Their page is still there.

File: README.md [hunk 18]
```diff
@@ -564,14 +470,10 @@ And thank you for your help!
...
 * [The Side Quest](http://www.thesidequestbar.com/) - local nerd bar, not exactly tech but lots of interesting events
 * [Observatory Park](https://www.geaugaparkdistrict.org/parks/observatorypark.shtml) - a Dark Sky park
-* [Flight by Launch League](http://www.launchleague.org/flight/)
-* [TechInvest](https://techinvest.wpengine.com/)
 * [GDEX](https://www.thegdex.com/) - "The Midwest's Premier Gaming Expo"
-* [N.E. Geek Expo](http://www.negeekexpo.com/)
 * [Kent Hack Enough](https://khe.io/) - KSU Hackathon
 * [Hack YSU](http://hackysu.com/) - YSU's annual hackathon
 * [HackerX](https://hackerx.org/) - the world’s largest invite-only networking & recruiting event for developers (one hosted in CL)
-* [NOARSfest](https://www.noars.net/hamfests/noarsfest/) presented by The Northern Ohio Amateur Radio Society
```
Comment (mrfright) [inline README.md:574]: Updated to this

https://www.noars.net/?page_id=137

File: README.md [hunk 19]
```diff
@@ -606,27 +502,20 @@ And thank you for your help!
...
 * [WorkinTech.io](http://workintech.io/)
-* [FirstDevJob](https://firstdevjob.com/)
 * [Free and Open Source Jobs](https://www.fossjobs.net/)
-* [JumpStart Jobs Twitter](https://twitter.com/jumpstart_jobs)
 * [JumpStart Jobs Site](http://www.jumpstartinc.org/talent/job-seekers/)
 * [Find.jobs](https://find.jobs/)
 * [Adafruit Jobs Board](https://jobs.adafruit.com/)
 * [JumpStart client job search](https://jumpstartinc.secure.force.com/careers)
 * [OhioMeansJobs](https://jobs.ohiomeansjobs.monster.com/Search.aspx)
-* [Ohio To Work](https://ohiotowork.com/)
```
Comment (mrfright) [inline README.md:622]: Can update to this instead

https://www.findyourohio.com/

File: README.md [hunk 20]
```diff
@@ -641,7 +530,6 @@ And thank you for your help!
 * [Underdog.io](https://underdog.io/)
 * [Remote Work Junkie](https://jobs.remoteworkjunkie.com/)
 * [Skip the Drive](https://www.skipthedrive.com/)
-* [WellFound](https://wellfound.com/jobs)
```
Comment (mrfright) [inline README.md]: This site is still there.

File: README.md [hunk 21]
```diff
@@ -656,12 +544,10 @@ And thank you for your help!
 
 ## Business Development
 
-* [NorTech](http://www.nortech.org/)
```
Comment (mrfright) [inline README.md:659]: Updated to 

https://northeastohioregion.com/

File: README.md [hunk 22]
```diff
@@ -671,21 +557,14 @@ And thank you for your help!
...
 * [OHTec](http://ohtec.org/), their [Twitter](https://twitter.com/OHTecOnline)
 * [GLIDE](http://www.glideit.org/)
-* [BioEnterprise](http://www.bioenterprise.com/)
 * [RITE - Regional Information Technology Engagement](https://ohiorite.com/)
-* [Conexus](http://www.conexus.jobs/)
-* [startup CLE](http://www.startupcle.co/)
 * [Launch League](http://www.launchleague.org/)
 * [VentureOhio](http://www.ventureohio.org/)
 * [TechOhio](https://weare.techohio.ohio.gov/)
-* [Ohio Third Frontier](https://development.ohio.gov/bs_thirdfrontier/)
```
Comment (mrfright) [inline README.md]: Site is still up.

File: README.md [hunk 23]
```diff
@@ -717,8 +592,6 @@ And thank you for your help!
 * [askadev](http://www.askadev.com/)
 * [LanceList](http://www.lancelist.com/#)
 * [Meeet](http://beta.meeet.co/)
-* [Zen99](https://zen99.co/)
-* [upwork](https://www.upwork.com/)
```
Comment (mrfright) [inline README.md:719]: upwork still there

**Review by mrfright — COMMENTED** [review]
made some comments of links that are still there or updated

File: README.md (hunk 14 above)
Comment (AddoSolutions) [reply README.md:343]: Weird a bunch of these came back as unresolved – thanks for scrubbing through them!

**Comment by mrfright** [comment]
Fantastic work. Thanks!

**Pull request merged**